### DISCUSSION TITLE
Add sync status page

### DISCUSSION THREAD

**Discussion by applejag** [thrillfall/nextcloud-gpodder#88] [Ideas]
Hello! I'd really like a status page to understand if the synced podcasts are actually synced.

I've already started implementing this and I am preparing a PR. I'm making this issue so that other don't accidentally do the same work. I dibs this task :)

@thrillfall any comments on how you'd see this implemented in particular? I'm currently implementing a very simple Vue page in the personal settings section. Also, would you even accept such a feature (considering there is no such status page already)? :)

**Comment by thrillfall** [comment]
I also pondered this idea but failed to find a reason to have this view. If you have a reason for it then don't hesitate to make it so 👍

**Comment by thrillfall** [comment]
I accept anything that does not break the sync functionally. If you think you need it then probably someone else will too.

As long as it is covered by tests that is.
